### DISCUSSION TITLE
Mack

### DISCUSSION THREAD

**Discussion by sebinthankachan7** [community/community#37003] [New to GitHub]
The difference between forking and cloning a repository on GitHub

**Comment by radar-blip** [comment]
#35849

**Comment by DenverCoder1** [comment]
**Fork** means to create your own copy of a repository on your GitHub account. This is something you would do when you want to contribute to an open source project you don't have write access to since you can write to *your* version of the repository and then open a pull request. You can also fork a project for personal use or to maintain a dead project since it allows to keep the commit history and add onto it yourself as a new repository.

**Clone** simply means to download a repository to your computer. It will copy all of the files from GitHub into your local folder. This does not create a new repository, rather it just allows you to have an existing repository locally. Unlike forking cloning does not require you to have a GitHub account.

**Comment by ossanna16** [comment]
Hi there @sebinthankachan7 and welcome to our community 😄. Thanks so much for asking a great question!
